PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/webapp/WebAppBasePropertyView.java

```diff
@@ -153,25 +167,52 @@ public void actionPerformedFunc(ActionEvent event) {
 
         lnkUrl.setHyperlinkText("<Loading...>");
         setTextFieldStyle();
+
+        AzureEventBus.after("webapp.start", this::onWebAppStatusChanged);
+        AzureEventBus.after("webapp.stop", this::onWebAppStatusChanged);
+        AzureEventBus.after("webapp.restart", this::onWebAppStatusChanged);
+        AzureEventBus.after("webapp.delete", this::onWebAppStatusChanged);
+
+        AzureEventBus.after("function.start", this::onWebAppStatusChanged);
+        AzureEventBus.after("function.stop", this::onWebAppStatusChanged);
+        AzureEventBus.after("function.restart", this::onWebAppStatusChanged);
+        AzureEventBus.after("function.delete", this::onWebAppStatusChanged);
+    }
+
+    private void onWebAppStatusChanged(IAppService app) {
+        if (StringUtils.equalsIgnoreCase(this.resourceId, app.id())) {
+            if (!app.exists()) {
+                closeEditor(app);
+                return;
+            }
+            presenter.onLoadWebAppProperty(app.subscription().getId(), this.resourceId, null);
```

Comment:
need to override this method for web app deployment slot view

Reply:
it is for webapp and functions